modules/olm-catalog-source-and-psa.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-custom-catalogs.adoc
+
+:_content-type: CONCEPT
+[id="olm-catalog-sources-and-psa_{context}"]
+= Catalog sources and pod security admission
+
+_Pod security admission_ was introduced in {product-title} 4.11 to ensure pod security standards. Catalog sources built using the SQLite-based catalog format and a version of the `opm` CLI tool released before {product-title} 4.11 cannot run under restricted pod security enforcement.
+
+In {product-title} {product-version}, namespaces do not have restricted pod security enforcement by default and the default catalog source security mode is set to `legacy`.  
+
+Default restricted enforcement for all namespaces is planned for inclusion in a future {product-title} release. When restricted enforcement occurs, the security context of the pod specification for catalog source pods must match the restricted pod security standard. If your catalog source image requires a different pod security standard, the pod security admissions label for the namespace must be explicitly set.
+
+[NOTE]
+====
+If you do not want to run your SQLite-based catalog source pods as restricted, you do not need to update your catalog source in {product-title} {product-version}.
+
+However, it is recommended that you take action now to ensure your catalog sources run under restricted pod security enforcement. If you do not take action to ensure your catalog sources run under restricted pod security enforcement, your catalog sources might not run in future {product-title} releases.
```

Comment:
@joelanford and @bparees: If customers do not take mitigation actions now, could their upgrade to OCP 4.13 (or whenever restricted enforcement occurs) be blocked? To put it another way, do I need to be more explicit and say: "you must update your catalog source to comply with restricted enforcement before updating OCP when restricted enforcement occurs."

Reply:
The upgrade to 4.13 won't be explicitly blocked (at least right now, i don't believe there is a plan to block upgrades based on the existence of pods that are throwing PSA warnings and will fail admission in 4.13 based on default restricted enforcement, though @deads2k  or @stlaz would be the ones to know what is planned there), but if they have a privileged pod running in a NS that starts being enforced as restricted in 4.13, that pod will not be able to be recreated (if for example it is rescheduled to a new node, or deleted for some reason, or, in the case of a catalog, recreated due to an updated catalog image)